debs-to-download

```diff
@@ -23,4 +23,5 @@ python3-revpimodio2
 revpipycontrol
 revpipyload
 revpicommander
-chromium
+epiphany-browser
+epiphany-browser-data
```

Comment:
epiphany-browser depends on epiphany-browser-data,  so only epiphany-browser should be sufficient.